tests/TestHelpers/GraphHelper.php

```diff
@@ -1893,4 +1893,59 @@ public static function getSharesSharedByMe(
 			self::getRequestHeaders()
 		);
 	}
+
+	/**
+	 * @param string $baseUrl
+	 * @param string $xRequestId
+	 * @param string $user
+	 * @param string $password
+	 * @param string $share
+	 *
+	 * @return ResponseInterface
+	 * @throws GuzzleException
+	 */
+	public static function acceptShare(
+		string $baseUrl,
+		string $xRequestId,
+		string $user,
+		string $password,
+		string $share
+	): ResponseInterface {
+		$response = self::getSharesSharedWithMe(
+			$baseUrl,
+			$xRequestId,
+			$user,
+			$password,
+		);
+		$responseBody = json_decode($response->getBody()->getContents(), true);
+		foreach ($responseBody["value"] as $value) {
+			if (isset($value["name"]) && $value["name"] === $share) {
+				$remoteId = $value["remoteItem"]["id"];
+				$permissionId = $value["remoteItem"]["permissions"][0]["id"];
+				$driveId = $value["parentReference"]["id"];
+			}
+		}
```

Comment:
could you implement share store for sharingNG shares? then we can get ids from the stored share